lib/generators/rolemodel/css/base/templates/app/views/layouts/partials/_flash.html.slim

```diff
@@ -0,0 +1,9 @@
+.container
+  - if notice
+    p.flash.flash--notice = notice
+  - if flash[:info]
+    p.flash.flash--info = flash[:info]
+  - if alert
+    p.flash.flash--alert = alert
+  - if flash[:warning]
+    p.flash.flash--warning = flash[:warning]
```

Comment:
I thought we had another PR recently that was adding this

Reply:
We could replace this with what we are doing in Fujifilm https://github.com/RoleModel/fujifilm_cimba/blob/master/app/views/shared/_flash.html.slim

And make the location of partial `app/views/shared/_flash.html.slim`